graphql_compiler/schema_generation/schema_graph.py

```diff
@@ -615,20 +615,21 @@ def _get_element_properties(self, class_name, non_link_property_definitions):
                                      u'more than once, this is not allowed!'
                                      .format(property_name, class_name))
 
-            graphql_type = self._get_graphql_type(class_name, property_definition)
-            default_value = _get_default_value(class_name, property_definition)
-            property_descriptor = PropertyDescriptor(graphql_type, default_value)
-            property_name_to_descriptor[property_name] = property_descriptor
+            maybe_graphql_type = self._try_get_graphql_type(class_name, property_definition)
+            if maybe_graphql_type:
```

Comment:
I think you specifically want to check against `is None` here, as opposed to checking for truthiness in general.